commit feature delete and insert user

namdhgc opened the pull request:
What i did:
 + Create function delete and insert data 
 + Create view for action delete and insert data
 + Create jQuery code for delete and insert event such as: jQuery validate, submit form using Ajax.

namdhgc committed:
commit freature delete user

jackdiff reviewed (changes requested):
Please remove unused jquery localization files.

namdhgc commented:
@jackdiff i deleted unused file, please recheck.

namdhgc committed:
[ feature delete and insert user ]:
 + create function delete and insert user
 + remove unused jquery localization files

jackdiff commented on app/Http/Controllers/UserController.php:
```diff
@@ -0,0 +1,225 @@
+class UserController extends Controller
+{
+
+    function __construct()
```
please remove empty function

namdhgc replied:
removed

jackdiff commented on app/Http/Controllers/UserController.php:
```diff
@@ -0,0 +1,225 @@
+
+                // image mime types start with "image/"
+                if( substr( $file_mime, 0, 5 ) == 'image' ) {
+
```
Remove empty line

jackdiff commented on app/Http/Controllers/UserController.php:
```diff
@@ -0,0 +1,225 @@
+        return $results;
+    }
+
+
```
Should use 1 empty line

namdhgc replied:
applied change

jackdiff commented on app/Http/Controllers/UserController.php:
```diff
@@ -0,0 +1,225 @@
+            ];
+
+            $deleted_record = $ModelUser->update_db( $data, $where );
+            // $deleted_record['meta']['success'] = true;
```
remove unsed code comment out

namdhgc replied:
removed

jackdiff commented on routes/web.php:
```diff
@@ -11,6 +11,41 @@
+
+		$results = App::make('App\Http\Controllers\UserController')->deleteData();
+
+		return json_encode( $results );
```
please move return data into controller

namdhgc replied:
applied change

namdhgc committed:
applied change as comment:
 + remove empty function, empty line
 + move return data into Controller

jackdiff commented on app/Http/Controllers/UserController.php:
```diff
@@ -0,0 +1,215 @@
+    */
+    public function getData()
+    {
+
```
Remove this empty line

jackdiff commented on app/Http/Models/User.php:
```diff
@@ -0,0 +1,225 @@
+					$query = $query->get();
+					break;
+			}
+			// $queries = DB::getQueryLog();
```
Remove these comments

namdhgc replied:
removed

jackdiff commented on app/Http/Controllers/UserController.php:
```diff
@@ -0,0 +1,215 @@
+            }
+        }
+
+        return json_encode( $results );
```
please use this convention: return response()->json($results);

namdhgc replied:
applied change

jackdiff commented on routes/web.php:
```diff
@@ -11,6 +11,31 @@
+	}]);
 });
+
+Route::group(['as' => 'post-'], function () {
```
please use this convention:
Route::group(['prefix' => 'user']), function() {
	Route::post('delete', 'UserController@deleteData');
	Route::post('add', 'UserController@insertData');
	Route::post('edit', 'UserController@updateData');
});

namdhgc replied:
applied change

namdhgc committed:
applied change as comment:
 + use new convention
 + removed comments

namdhgc committed:
applied change as comment:
 + remove empty line
 + change position of curly brackets
